### PR TITLE
Generators

### LINKED ISSUE #32: Add blueprint for components and helpers
After #27 is merged it would be great to add a couple of additional blueprints to ease the creation of reusable components and helpers.

__1. page-object-component__

```sh
$ ember generate page-object-component navbar

installing
  create tests/pages/components/navbar.js
```

which generates...

```js
import PageObject from '../page-object';

export default PageObject.component({
});
```

__2. page-object-helper__

```sh
$ ember generate page-object-helper checked

installing
  create tests/pages/helpers/checked.js
```

```js
import PageObject from '../page-object';

function checked(selector, options) {
  return "a value";
}

export default PageObject.customHelper(checked);
```

### PR DESCRIPTION
**Pull request by juanazam**
Closes #32

### CODE REVIEW

File: blueprints/page-object-helper/files/tests/pages/helpers/__name__.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import PageObject from '../../page-object';
+
+function checked(selector /*, options */) {
+  return $(selector).prop('checked');
```
Comment (san650) [inline blueprints/page-object-helper/files/tests/pages/helpers/__name__.js]: Don't assume `$` is global, maybe we can import `Ember`

```js
import Ember from 'ember';

let $ = Ember.$;

...
```

**Comment by san650** [comment]
It would be great to have the list of available blueprints in the [README.md](https://github.com/san650/ember-cli-page-object#blueprints)

Maybe something like:

----

The addon includes the following blueprints

| Name | Description |
| -------- | --------------- |
| `page-object` | Creates a new page object under `tests/pages` folder |
| `page-object-component` | Creates a new a component object to be used on a page object under `tests/pages/components` folder. |
| `page-object-helper` |  Creates a new a helper object to be used on a page object under `tests/pages/helpers` folder. |

----

And then leave the current `ember generate page-object users` example. I don't think we need to add examples for the others.

File: blueprints/page-object-helper/files/tests/pages/helpers/__name__.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+}
+
+export default PageObject.customHelper(checked);
```
Comment (san650) [inline blueprints/page-object-helper/files/tests/pages/helpers/__name__.js]: We can use the name provided by the user as the name of the function:

```js
function <%= camelizedModuleName %>(selector/*, options*/) {
  return ...;
}

export default PageObject.customHelper(<%= camelizedModuleName %>);
```

What do you think?

http://www.ember-cli.com/extending/#template-variables-aka-locals

File: blueprints/page-object-helper/files/tests/pages/helpers/__name__.js (hunk 1 above)
Comment (juanazam) [inline blueprints/page-object-helper/files/tests/pages/helpers/__name__.js]: :+1:

File: blueprints/page-object-helper/files/tests/pages/helpers/__name__.js (hunk 2 above)
Comment (juanazam) [inline blueprints/page-object-helper/files/tests/pages/helpers/__name__.js]: :+1:

**Comment by san650** [comment]
@juanazam ping

**Comment by juanazam** [comment]
@san650 pong

**Pull request merged**